conf/app.ini

```diff
@@ -443,7 +443,8 @@ PULL = 300
 GC = 60
 
 [mirror]
-; The default interval in hours for fetching updates.
+; Defines the default interval (in hours) until the next synchronization for a mirror (after a successful mirror sync)
+; Can be overridden individually for each mirror in the mirror settings
```

Comment:
```suggestion
; Defines the default interval (in hours) until the next sync for a mirror (after a successful mirror sync).
; It can be overridden individually for each mirror repository in the settings.
```